zerver/tests/test_auth_backends.py

```diff
@@ -7999,6 +7999,53 @@ def test_ldap_group_sync(self) -> None:
                 'WARNING:django_auth_ldap:search_s("ou=groups,dc=zulip,dc=com", 1, "(&(objectClass=groupOfUniqueNames(uniqueMember=uid=cordelia,ou=users,dc=zulip,dc=com))", "None", 0) while authenticating cordelia',
             ],
         )
+        with (
```

Comment:
Added a comment on top, folding it into the code above is possible but it involved changing the assertions quite much since sync ldap descriptions will be called multiple times. Could do it if it is something you really want

Reply:
Yeah a comment is fine i think.